### ISSUE TITLE
Invalid certificate created for front-proxy-ca

### ISSUE THREAD

**Issue by lulf** [openshift/origin#19878]
The certificate generated by 'oc cluster up' in 'front-proxy-ca.crt' is not a valid x509 certificate. This prevents the certificate to be trusted by a x509-compliant implementation, such as Java keystore. This certificate is in turn used in the 'extension-apiserver-authentication' configmap in the field 'requestheader-client-ca-file'. 

According to https://tools.ietf.org/html/rfc5280#section-4.1.2.4 

"The issuer field MUST contain a non-empty distinguished name (DN)."

##### Version
```
$ oc version
oc v3.9.0+191fece
kubernetes v1.9.1+a0ce1bc657
features: Basic-Auth GSSAPI Kerberos SPNEGO
```

##### Steps To Reproduce
1. `oc cluster up`
2. `oc login -u system:admin`
3. `oc get configmap -n kube-system extension-apiserver-authentication -o jsonpath={.data.requestheader-client-ca-file} > ca.crt`
4. `keytool -import -noprompt -file ca.crt -alias firstCA -deststorepass 123456 -keystore truststore.jks`

##### Current Result

```
keytool error: java.lang.Exception: Input not an X.509 certificate
```

##### Expected Result

```
Certificate was added to keystore
```

##### Additional Information

After digging into this issue in the 3.9 source tree, it looks like there are 2 CAs created for the front-proxy.
front-proxy-ca.crt and frontproxy-ca.crt.

**Comment by mrogers950** [comment]
@lulf front-proxy-ca.crt is no longer created in 3.10 cluster up. It was created separately from frontproxy-ca.crt and other master certs and I don't think it was referenced in the default config.

**Comment by lulf** [comment]
@mrogers950 Would it be possible to have the fix in #19880 (or another fix) backported to 3.9? Also, is there a 3.10 nightly/snapshot that I can use to verify/test that it works?

**Comment by mrogers950** [comment]
@lulf I checked on master (not sure about a proper nightly repo) and the extension-apiserver-authentication configmap contains the valid frontproxy-ca cert in 3.10, so a more correct fix for 3.9 would probably be to update the configmap with frontproxy-ca.crt.

**Comment by mrogers950** [comment]
@lulf Ah, I see #19880 does just that.
